analyzer/src/funTest/assets/projects/synthetic/conan-expected-output-txt.yml

```diff
@@ -35,11 +35,11 @@ packages:
 - id: "Conan::libcurl:7.79.1"
   purl: "pkg:conan/libcurl@7.79.1"
   declared_licenses:
-  - "MIT"
+  - "curl"
```

Comment:
I'm wondering how this ever showed `MIT` as this was always wrong :/

Reply:
See https://github.com/conan-io/conan-center-index/pull/12568.